pages/success.tsx

```diff
@@ -18,6 +19,7 @@ dayjs.extend(timezone);
 export default function Success(props) {
     const router = useRouter();
     const { location } = router.query;
+    const { name } = router.query;
```

Comment:
You _could_ combine line 21 and 22 to:
`const { location, name } = router.query;`

Reply:
But the rest is neat, so I'll already approve it. :)

Reply:
done